docs/src/tutorial.md

```diff
@@ -121,5 +121,35 @@ DI.jacobian(h, prep, backend, x)
 
 ## Mooncake.jl API
 
+### Terminology Comparison with DifferentiationInterface
+
+Mooncake.jl uses terminology rooted in mathematical analysis, specifically Frechet derivatives and their adjoints, as described in detail in [Algorithmic Differentiation](@ref). This differs somewhat from the conventions used by [DifferentiationInterface.jl](https://github.com/JuliaDiff/DifferentiationInterface.jl) and other AD packages.
+
+The following table shows the correspondence between terminology:
+
+| **Mode** | **General Case** | **Special Case** | **Mooncake Function** |
+|----------|------------------|------------------|----------------------|
+| **Forward mode** | **Frechet derivative** | **directional derivative** | N/A |
+| | *(DI: pushforward)* | *(DI: derivative)* | |
```

Comment:
`DI.derivative` is not a directional derivative, it is only applicable when the input variable is a scalar, i.e. $f(t)$ with $t \in \mathbb{R}$. In this regard, it is the symmetric of the gradient. So I would rather name this special case "derivative"

Reply:
Also, the Mooncake forward-mode function is available, it is called `value_and_derivative!!` (not sure why isn't documented)

Reply:
I don't think that is true, `Mooncake.value_and_derivative!!` is exactly a pushforward (which is why I wanted to name it that way)